samples: nrf9160: modem_shell: Add 'location' command (#6086)
nrfconnect/sdk-nrf: samples/nrf9160/modem_shell/README.rst

----

> Locaton tool
> ============

Reviewer: ```suggestion
Location tool
=============
```

MoSh command: ``location``

> You can use the Location tool for retrieving devices location with different location methods.

Reviewer: ```suggestion
You can use the Location tool for retrieving device's location with different methods.
```

> See :ref:`lib_location` library for information on the configuration of different location methods and services
> but there are some available by default to ease trials.

Reviewer 2: ```suggestion
See :ref:`lib_location` library for information on the configuration of different location methods and services.
Default configurations are available to facilitate trials.
```

Author: Pekka made little modification to your suggestion so I accepted that.

> See :ref:`lib_location` library for information on the configuration of different location methods and services
> but there are some available by default to ease trials.

Reviewer: ```suggestion
See :ref:`lib_location` library for information on the configuration of different location methods and services.
Some default configurations are available to facilitate trials.
```

Examples
--------

* Retrieve location with default configuration:

  .. code-block:: console

     location get

> * Retrieve location with Wi-Fi positioning (you need to have Wi-Fi build and chip) and if not found, use cellular positioning:

Reviewer: ```suggestion
* Retrieve location with Wi-Fi positioning (you need to have a Wi-Fi-enabled device) and if not found, use cellular positioning:
```

Author: This changes the meaning. I'm trying to say that you need to have the device as well as MOSH built with Wi-Fi support

Reviewer: How about something like: "You need to have a Wi-Fi enabled device and build the sample Wi-Fi support."

Author: That's fine although a bit long in the parenthesis. Can you modify the suggestion?

Reviewer: Did that.

> * Retrieve location with Wi-Fi positioning (you need to have Wi-Fi build and chip) and if not found, use cellular positioning:

Author: I've used Pekka'ssuggestion here.
